src/lib/sql/gdfs_cache.hpp

```diff
@@ -87,6 +87,19 @@ class GDFSCache : public AbstractCache<Key, Value> {
     this->_capacity = capacity;
   }
 
+  void resize(size_t capacity) {
+    if (_queue.size() > capacity) {
+      for (size_t difference = 0; difference < _queue.size() - capacity; ++difference) {
```

Comment:
Couldn't you do this in one line with `while (_queue.size > capacity)`?

Reply:
Done